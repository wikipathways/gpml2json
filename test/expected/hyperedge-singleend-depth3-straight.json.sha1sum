2eda6cbf2497133f8e6655d5a8be2a5d67d76ac2 ?-
