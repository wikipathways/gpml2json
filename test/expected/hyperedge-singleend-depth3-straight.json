{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["idaf905386","id28382a1c","idc088e267","id6d7f3bae","faa53","dde22","d93ba"],"drawAs":"rect","gpmlElementName":"Pathway","height":372.4925040280843,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":447.7009507346585,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130715","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/329745a41163454cfef46adc5d0627ae6797bade4b6c1a614cf385a0862b0646"},"entityMap":{"faa53":{"id":"faa53","textContent":"Catalyzer","wpType":"GeneProduct","x":193.22556206451424,"y":322.4925040280843,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"dde22":{"id":"dde22","textContent":"Inhibitor1","wpType":"GeneProduct","x":337.7009507346585,"y":181.01123595505615,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d93ba":{"id":"d93ba","textContent":"Inhibitor2","wpType":"GeneProduct","x":256.94900605012964,"y":261.019878997407,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32769,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"idaf905386":{"id":"idaf905386","borderWidth":1,"zIndex":12288,"points":[{"x":160.09831016073218,"y":151.36848170556027},{"x":293.43164349406544,"y":151.36848170556024}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","burrs":["e0874"]},"e0874":{"id":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idaf905386","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id28382a1c":{"id":"id28382a1c","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"faa53","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":233.22556206451424,"y":322.4925040280843,"orientation":[0,-1]},{"isAttachedTo":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[],"relativeOffset":[0,0]},"x":234.99542950229596,"y":151.36848170556024,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["faa53","idaf905386"],"burrs":["e4378"]},"e4378":{"id":"e4378","attachmentDisplay":{"position":[0.7632149127793446,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"id28382a1c","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"idc088e267":{"id":"idc088e267","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"dde22","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":337.7009507346585,"y":191.01123595505615,"orientation":[-1,0]},{"isAttachedTo":"e4378","attachmentDisplay":{"position":[0.7632149127793446,0],"offset":[],"relativeOffset":[0,0]},"x":234.5763512866718,"y":191.88809825674846,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimInhibition","isAttachedTo":["dde22","id28382a1c"],"burrs":["acbce"]},"acbce":{"id":"acbce","attachmentDisplay":{"position":[0.4,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idc088e267","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id6d7f3bae":{"id":"id6d7f3bae","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"d93ba","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":296.94900605012964,"y":261.019878997407,"orientation":[0,-1]},{"isAttachedTo":"acbce","attachmentDisplay":{"position":[0.4,0],"offset":[],"relativeOffset":[0,0]},"x":296.45111095546383,"y":191.36198087573308,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimInhibition","isAttachedTo":["d93ba","idc088e267"]}}}
